src/BenchmarkDotNet/Toolchains/NativeAot/Generator.cs

```diff
@@ -143,6 +143,7 @@ private string GenerateProjectForNuGetBuild(BuildPartition buildPartition, Artif
     <EnsureNETCoreAppRuntime>false</EnsureNETCoreAppRuntime> <!-- workaround for 'This runtime may not be supported by.NET Core.' error -->
     <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles> <!-- workaround for 'Found multiple publish output files with the same relative path.' error -->
     <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings> <!-- fix System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.0 -->
```

Comment:
this project file should never be used for such old tfms (because NativeAOT was not supported back then)

Reply:
Same as other comments.